packages/data/addon/models/metadata/table.js

```diff
@@ -58,14 +58,15 @@ let Model = EmberObject.extend({
     if (timeGrains) {
       this.set(
         'timeGrains',
-        timeGrains.map(timeGrain => {
-          let timeGrainPayload = assign({}, timeGrain, { source: this.source }),
-            owner = getOwner(this);
-          return owner.factoryFor('model:metadata/time-grain').create(timeGrainPayload);
-        })
-        .sort((l, r) => {
-          return (timeGrainSorting[l.name] || Infinity) - (timeGrainSorting[r.name] || Infinity);
-        })
+        timeGrains
```

Comment:
CAn we revert? This file is not changed.

Reply:
The commit formatting command is reverting it back to this every time :/